exonum-light-client/src/test/java/com/exonum/client/ExplorerApiHelperTest.java

```diff
@@ -43,4 +40,103 @@ void parseSubmitTxResponse() {
     HashCode actual = ExplorerApiHelper.parseSubmitTxResponse(json);
     assertThat(actual, equalTo(HashCode.fromString(expected)));
   }
+
+  @Test
+  void parseGetTxResponseInPool() {
+    String json = "{\n"
+        + "    'type': 'in-pool',\n"
+        + "    'content': {\n"
+        + "        'debug': {\n"
+        + "            'to': {\n"
+        + "                'data': []\n"
+        + "            },\n"
+        + "            'amount': 10,\n"
+        + "            'seed': 9587307158524814255\n"
+        + "        },\n"
+        + "        'message': 'ba76da2363c062f3dc25f5e353d0f8de2e3d02896b9c8f"
+        + "01683dbde7e3194eb90000800000000a220a20f2279c2df559fc3a8fae58c44ce1d"
+        + "fceb17d3f035a2480d54625f3e7ae8b5197100a18afc7b9838aa2bd8685019a6bf7a"
+        + "720fdfdf27948330d0a451f35aef3eb64f80f568136be91d2667f75b7eb4b03ba279b"
+        + "70c69bbf5b69db8d8edf5e1049f51c86e937c10d1408374faa08'\n"
+        + "    }\n"
+        + "}";
+    TransactionResponse transactionResponse = ExplorerApiHelper.parseGetTxResponse(json);
+
+    assertThat(transactionResponse.getStatus(), is(TransactionStatus.IN_POOL));
+    assertThat(transactionResponse.getMessage(), notNullValue());
```

Comment:
Shan't we extract the message as a constant and assert that we get exactly that?